app/views/foreman_kernel_care/job_templates/install_kernelcare.erb

```diff
@@ -4,7 +4,7 @@ name: LivePatching - Install Kernelcare
 model: JobTemplate
 job_category: LivePatching - Ansible Default
```

Comment:
Oh no, this should have been
job_category: LivePatching - Script Default

Reply:
Please create a new PR and unsure that everything is alright and we don't miss anything before I create a new version